mappings/cvss_v3.json

```diff
@@ -727,11 +727,15 @@
           "children": [
             {
               "id": "privilege_escalation",
-              "cvss_v3": "AV:N/AC:H/PR:N/UI:R/S:U/C:N/I:L/A:N"
+              "cvss_v3": "AV:L/AC:H/PR:N/UI:N/S:U/C:L/I:L/A:N"
+            },
+            {
+              "id": "non_default_folder_privilege_escalation",
+              "cvss_v3": "AV:L/AC:H/PR:L/UI:R/S:U/C:L/I:L/A:N"
```

Comment:
Regarding an `AC:H` Does this have to be the case here. I get that it does require some information however, the example that I had looked into was mainly talking about setups being saved to directories that are not normal. Like the user is given the choice to use different directories. To me that seems to mean that the AC should not be set to High. I feel like I'm not understanding something.

Reply:
These are the CVSS definitions for Attack Complexity:

High: "A successful attack depends on conditions beyond the attacker's control. That is, a successful attack cannot be accomplished at will, but requires the attacker to invest in some measurable amount of effort in preparation or execution against the vulnerable component before a successful attack can be expected. For example, a successful attack may require the attacker: to perform target-specific reconnaissance; to prepare the target environment to improve exploit reliability; or to inject herself into the logical network path between the target and the resource requested by the victim in order to read and/or modify network communications (e.g. a man in the middle attack)."

Low: "Specialized access conditions or extenuating circumstances do not exist. An attacker can expect repeatable success against the vulnerable component."

All of the proposed variants have set those to high, as the average attack scenario seems to require a good amount of effort on an unpatched environment